antenna-documentation/scripts/createOrUpdateVersionDocumentationFolder.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Copyright (c) Bosch Software Innovations GmbH 2019.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+BASEDIR=$(readlink -f "$0")
+BASEDIR=$(dirname "$BASEDIR")
+
+set -e
+
+CURRENT_BRANCH=$1
+# checkout branch and build antenna
+git checkout ${CURRENT_BRANCH}
+mvn clean install -DskipTests
+mvn site -f antenna-documentation/pom.xml
+
+cp -R antenna-documentation/target/documentation/ temp_documentation/
+
+git checkout gh-pages
+
+rsync -av temp_documentation/ ${CURRENT_BRANCH}/
```

Comment:
Why first copy to a temporary location? The `git checkout gh-pages` shouldn't tamper with the untracked `antenna-documentation/target/documentation/` directory, so we should be able to directly copy from there. Also, why once use `cp` and once `rsync` to copy directories? I'd prefer to align on either one for consistency.